lib/agent0/examples/lp_agents.py

```diff
@@ -0,0 +1,109 @@
+"""Script to showcase running default implemented agents."""
```

Comment:
would it be possible to add the lp_and_arb bot to hyperdrive_agents? I'd rather that than creating a new example file.

Reply:
good idea